id-repository/id-repository-salt-generator/pom.xml

```diff
@@ -18,8 +18,8 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
```

Comment:
changed the pom version 1.2.1-SNAPSHOT to 1.2.1-java21-SNAPSHOT

Reply:
Changed version to java21 snapshot